include/boost/geometry/algorithms/detail/overlay/traversal_switch_detector.hpp

```diff
@@ -565,10 +591,83 @@ struct traversal_switch_detector
         }
     }
 
+#if defined(BOOST_GEOMETRY_DEBUG_TRAVERSAL_SWITCH_DETECTOR)
+    void debug_show_results()
```

Comment:
this shows quite a neat list of results and I would like to keep it.

For example
```
BEGIN SWITCH DETECTOR (region_ids and isolation) REVERSE_2
 ADD (s:0, m:0) TO REGION 1
 ADD (s:1, m:0) TO REGION 2
 ADD (s:1, m:1) TO REGION 3
END SWITCH DETECTOR

REGION 1 multiple [turns 0 1 2] regions { 2 : via [ 0 1 2 ] }
REGION 2 yes [turns 0 1 2 3] regions { 1 : via [ 0 1 2 ] } { 3 : via [ 3 ] }
REGION 3 yes [turns 3] regions { 2 : via [ 3 ] }

II 0 (50, 70) ->  [0/1] (1 false) / (2 false)
II 1 (70, 70) ->  [0/1] (1 false) / (2 false)
II 2 (70, 50) ->  [0/1] (1 false) / (2 false)
II 3 (50, 50) ->  [1/1] (2 true) / (3 true)
Analyze II from turn 0 (50, 70) region 1 -> to turn 1 (70, 70) region 2
Analyze II from turn 0 (50, 70) region 2 -> to turn 3 (50, 50) region 2
Analyze II from turn 3 (50, 50) region 2 -> to turn 2 (70, 50) region 1
Analyze II from turn 1 (70, 70) region 1 -> to turn 2 (70, 50) region 2
```